Firmware/tmc2130.cpp

```diff
@@ -73,15 +67,19 @@ uint16_t tmc2130_sg_cnt[4] = {0, 0, 0, 0};
 bool tmc2130_sg_change = false;
 
 
+//STEALTHCHOP REGISTER CONFIGURATION
+uint8_t tmc2130_pwm_ampl[4] = TMC2130_PWM_AMPL;     //pwm_ampl  
+uint8_t tmc2130_pwm_grad[4] = TMC2130_PWM_GRAD;     //pwm_grad
+uint8_t tmc2130_pwm_auto[4] = TMC2130_PWM_AUTO;     //pwm_auto
+uint8_t tmc2130_pwm_freq[4] = TMC2130_PWM_FREQ;     //pwm_freq
+
+uint8_t tmc2130_wave_fac[4] = {0, 0, 0, 0};
+
 bool skip_debug_msg = false;
 
 #define DBG(args...) printf_P(args)
```

Comment:
Were the changes in this block intentional?

Reply:
Yes, these changes were to bring the PWM settings in line with the format defined in the configuration file.